### ISSUE TITLE
EPIP: Adapt 5.5 (and eventually 5.6) to the new part 6

### ISSUE THREAD

**Issue by ue71603** [NeTEx-CEN/NeTEx-Profile-EPIP#3]
With EPIAP the information in EPIP about accessibility should be reduced to the minimum and mostly contain a reference to part 6.
At least a check for conformance must be made.

**Comment by ue71603** [comment]
From Christophe: You already can do it thanks to the NoticedObjectRef in NoticeAssignment (that can be a standalone element)... but that's true that this is not explicit and some words about it would help
